### PR TITLE
*: cleanup usage of testutil.RowsWithSep 

### LINKED ISSUE #32858: Clean up usage of testutil.RowsWithSep

### PR DESCRIPTION
**Pull request by tisonkun**
### What problem does this PR solve?

Issue Number: close #32858 

```release-note
None
```

### CODE REVIEW

**Review by hawkingrei — APPROVED** [review]

**Comment by hawkingrei** [comment]
@wjhuang2016  PTAL

**Review by lcwangchao — APPROVED** [review]

File: ddl/column_type_change_test.go [hunk 1]
```diff
@@ -2244,53 +2244,21 @@ func TestChangeFromTimeToYear(t *testing.T) {
...
-	tbl := tk.GetTableByName("test", "t")
-	require.NotNil(t, tbl)
-	require.Equal(t, 1, len(tbl.Cols()))
-
-	hook := &ddl.TestDDLCallback{Do: dom}
-	var (
-		checkErr1 error
-		checkErr2 error
-	)
-	hook.OnJobRunBeforeExported = func(job *model.Job) {
```
Comment (wjhuang2016) [inline ddl/column_type_change_test.go:2275]: Why do you remove the hook?

Comment (tisonkun) [reply ddl/column_type_change_test.go:2275]: Let me try to revert this change. According to the comment of the test, either test with ddl hook or serially execute sql statement should be the same.

Comment (tisonkun) [reply ddl/column_type_change_test.go:2275]: Done by 893d865a441ae4708b8ac28ffd62db04faaa643a

**Pull request merged**